src/cli/rustup_mode.rs

```diff
@@ -805,6 +805,9 @@ fn update(cfg: &mut Cfg, m: &ArgMatches<'_>) -> Result<()> {
         cfg.set_profile_override(p);
     }
     let cfg = &cfg;
+    if cfg.get_profile()? == Profile::Complete {
+        warn!("downloading with complete profile isn't recommneded unless you are a developer of the rust language")
```

Comment:
You have a typo here:
```suggestion
        warn!("downloading with complete profile isn't recommended unless you are a developer of the rust language")
```